### PR TITLE
Pre-release v2.2.0 - Documentation and DFU updates

### PR DESCRIPTION
**Pull request by lucianomartin**
Part of https://xmosjira.atlassian.net/browse/AP-15

### CODE REVIEW

File: examples/low_power_ffd/README.md
Comment (mbanth) [inline examples/low_power_ffd/README.md:1]: Has anyone checked that the `xrun` and `xgdb` instructions produce the expected results?  See my similar comment for the FFVA read me file.

File: examples/speech_recognition/README.md [hunk 1]
```diff
@@ -68,13 +68,7 @@ Run the following commands in the build folder to flash the model:
...
 Run the following command in the build folder to run the firmware.
 
-On Linux and Mac run:
-
-    make run_example_asr
-
-On Windows run:
-
-    nmake run_example_asr
+    xrun --xscope-realtime --xscope-port localhost:12345 example_asr.xe
```
Comment (mbanth) [inline examples/speech_recognition/README.md:71]: Another `xrun` instruction that I think someone should bench test.

File: doc/programming_guide/low_power_ffd/deploying/linux_macos.rst
Comment (mbanth) [inline doc/programming_guide/low_power_ffd/deploying/linux_macos.rst:1]: It looks like the low-power FFD example does not support use of the Cyberon ASR engine and model.  Can someone please confirm which examples do support the use of the Cyberon ASR engine?  Thanks.

**Review by mbanth — COMMENTED** [review]
All of the changes look good.  I have asked a few questions.

File: doc/programming_guide/low_power_ffd/deploying/linux_macos.rst
Comment (lucianomartin) [reply doc/programming_guide/low_power_ffd/deploying/linux_macos.rst:1]: Only the FFD example supports Cyberon ASR engine; @tomblackie-xmos confirmed that we don't need to provide an example of low power FFD with Cyberon: a potential customer can port the changes from the FFD example with Cyberon into the low power FFD example, if they want.

Comment (mbanth) [reply doc/programming_guide/low_power_ffd/deploying/linux_macos.rst:1]: Thanks for this information, @lucianomartin.

File: examples/ffva/README.md
[the remark this answers is not in the document]
Comment (lucianomartin) [reply examples/ffva/README.md:1]: All the builds are working on Windows.

File: examples/low_power_ffd/README.md
Comment (lucianomartin) [reply examples/low_power_ffd/README.md:1]: I tested them on Windows.

File: examples/speech_recognition/README.md (hunk 1 above)
Comment (lucianomartin) [reply examples/speech_recognition/README.md:110]: Tested on Windows.

**Pull request merged**